src/main/java/sirius/biz/jdbc/DatabaseController.java

```diff
@@ -129,6 +137,51 @@ public void executeQuery(WebContext ctx, JSONStructuredOutput out) throws SQLExc
         }
     }
 
+    /**
+     * Exports the given SQL query
+     *
+     * @param webContext the current request
+     */
+    @Permission(TenantUserManager.PERMISSION_SYSTEM_ADMINISTRATOR)
+    @Routed("/system/sql/export")
+    public void exportQuery(WebContext webContext) {
+        if (!webContext.isSafePOST()) {
+            throw Exceptions.createHandled().withSystemErrorMessage("Unsafe or missing POST detected!").handle();
+        }
+
+        String database = webContext.get("db").asString(defaultDatabase);
+        String sqlStatement = webContext.get("exportQuery").asString();
```

Comment:
Die ganzen Params waren in nem Zwischencommit als Konstante rausextrahiert, hatte mich gewundert wieso die wieder geinlined wurden :)